src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-29 calendar--start-day-wed">
+      <span class="calendar__day"></span>
```

Comment:
If you have no content, you can use a single tag, but this is only a recommendation for the future.
See example 
```suggestion
      <span class="calendar__day" />
```